frontend/src/metabase/admin/settings/components/SettingsLdapForm.tsx

```diff
@@ -166,9 +166,9 @@ export const SettingsLdapFormView = ({
               <FormTextInput {...fields["ldap-port"]} />
               <FormRadioGroup {...fields["ldap-security"]}>
                 <Group mt={"xs"}>
-                  <Radio value="none" label="None" />
-                  <Radio value="ssl" label="SSL" />
-                  <Radio value="starttls" label="StartTLS" />
+                  <Radio value="none" label={t`None`} />
+                  <Radio value="ssl" label={t`SSL`} />
+                  <Radio value="starttls" label={t`StartTLS`} />
```

Comment:
(not sure if we need to localize technical terms like SSL; I'm not even sure that's localizable)

Reply:
Let's localize the string and allow the translators to decide. A context string clarifying that it is short for "Secure Sockets Layer" would be helpful.